draft-ietf-sedate-datetime-extended.md

```diff
@@ -435,6 +427,49 @@ required to reject or perform some other error handling when
 encountering inconsistent or unrecognized suffix tags marked as
 critical.
 
+## Offset vs. Time Zone Conflicts
+
+Time zone offsets in a timestamp string can conflict with its time zone suffix.
+For example, a calendar application could store an IXDTF string representing a
+far-future meeting in a particular time zone. If that time zone's definition is
+subsequently changed to abolish Daylight Saving Time, IXDTF strings that were
+originally correct may now be conflicting.
+
+If time zone and offset are in conflict an application MUST (if the critical
+flag is used on the time zone hint) or MAY (if the critical flag is not used)
+reject the timestamp or resolve the conflict via user and/or programmer input.
+
+## Use of `Z` Offset Designator with Time Zone Suffix
+
+RFC 3339 (as well as this standard that extends it) provides the offset `-00:00`
+to indicate that the actual local time and local offset are unknown. However,
+many implementations use `Z` for this purpose, often to maximize compatibility
+with ISO 8601 which disallows `-00:00`. Therefore, implementations MAY choose to
+interpret `Z` the same as `-00:00`: exempting `Z` offsets from conflict
+detection and instead using the time zone suffix to determine the local offset
+and local time. Because of this common behavior, implementations SHOULD use
+`+00:00` instead of `Z` when recording a timestamp when the local UTC offset is
+known to be zero and SHOULD NOT use `+00:00` when the local UTC offset is
+unknown. 
+
+For example, the following strings represent timestamps at 00:14:07 local time
```

Comment:
Depending on the tolerance people have for forward references, even this set of examples could be used in the "update 3339" section.  It would be a bit redundant with the examples that explain conflicts in general and the critical flag, but I'm all in favor of trading redundancy for clarity.

Reply:
Good question. From a reading-order perspective, it does seem a bit weird to be talking about changes to 3339 before explaining what the changes are. Would it be more understandable for readers to explain the concepts in the new standard first (suffixes, critical flag, conflicts, etc.) and only then talk about what changes are needed to 3339?